src/index.ts

```diff
@@ -97,6 +100,14 @@ if (!USER_FUNCTION) {
 
 const SERVER = getServer(USER_FUNCTION!, SIGNATURE_TYPE!);
 const ERROR_HANDLER = new ErrorHandler(SERVER);
+
+if (DRY_RUN) {
+  console.log(`Function: ${TARGET}`);
+  console.log(`URL: http://localhost:${PORT}/`);
+  console.log('Dry run successfully, shutting down.');
+  process.exit(0);
```

Comment:
You could potentially move the process.exit(0) inside the SERVER.listen and take advantage of the console.logs there, but either is fine.